### PR TITLE
Update ipc-channel

### PR DESCRIPTION
**Pull request by SimonSapin**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2962)

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1663,9 +1721,12 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 "checksum core-text 10.0.0 (registry+https://github.com/rust-lang/crates.io-index)" = "81f59bff773954e5cd058a3f5983406b52bec7cc65202bef340ba64a0c40ac91"
 "checksum core-text 11.0.0 (registry+https://github.com/rust-lang/crates.io-index)" = "157ff38a92496dc676ce36d9124554e9ac66f1c1039f952690ac64f71cfa5968"
 "checksum crossbeam 0.2.12 (registry+https://github.com/rust-lang/crates.io-index)" = "bd66663db5a988098a89599d4857919b3acf7f61402e61365acfd3919857b9be"
+"checksum crossbeam-channel 0.2.4 (registry+https://github.com/rust-lang/crates.io-index)" = "6c0a94250b0278d7fc5a894c3d276b11ea164edc8bf8feb10ca1ea517b44a649"
 "checksum crossbeam-deque 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)" = "f739f8c5363aca78cfb059edf753d8f0d36908c348f3d8d1503f03d8b75d9cf3"
 "checksum crossbeam-epoch 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)" = "927121f5407de9956180ff5e936fe3cf4324279280001cd56b669d28ee7e9150"
+"checksum crossbeam-epoch 0.5.2 (registry+https://github.com/rust-lang/crates.io-index)" = "30fecfcac6abfef8771151f8be4abc9e4edc112c2bcb233314cafde2680536e9"
```
Comment (kvark) [inline Cargo.lock]: duplicate crossbeam dependencies detected

File: Cargo.lock [hunk 2]
```diff
@@ -1749,6 +1810,8 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 "checksum quote 0.6.3 (registry+https://github.com/rust-lang/crates.io-index)" = "e44651a0dc4cdd99f71c83b561e221f714912d11af1a4dff0631f923d53af035"
 "checksum rand 0.3.22 (registry+https://github.com/rust-lang/crates.io-index)" = "15a732abf9d20f0ad8eeb6f909bf6868722d9a06e1e50802b6a70351f40b4eb1"
 "checksum rand 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)" = "eba5f8cb59cc50ed56be8880a5c7b496bfd9bd26394e176bc67884094145c2c5"
+"checksum rand 0.5.5 (registry+https://github.com/rust-lang/crates.io-index)" = "e464cd887e869cddcae8792a4ee31d23c7edd516700695608f5b98c67ee0131c"
```
Comment (kvark) [inline Cargo.lock]: we, we have 3 different rand versions linked to

File: Cargo.lock (hunk 1 above)
Comment (SimonSapin) [reply Cargo.lock]: Yes, it looks like Rayon decided not to upgrade: https://github.com/servo/servo/pull/21325#issuecomment-410082199

File: Cargo.lock (hunk 2 above)
Comment (Eijebong) [reply Cargo.lock]: I thought I deduped rand some time ago...

**Pull request merged**